README.md

```diff
@@ -11,18 +11,18 @@ In some cases, you do want your app to be indexed - but only in your production
 Tested against Django 1.8, 1.9, 1.10, 1.11 on Python 2.7, 3.4 and 3.6
 
 ### Installation
-    
+
 Install from PIP
 
     pip install django-simple-robots
 
 In your root urlconf, add an entry as follows:
 
     from django.conf.urls import url
-    from simple_robots.views import serve_robots
```

Comment:
No need for all this API change stuff. Just put

```
serve_robots = ServeRobotsView.as_view()
```

at the end of the `views.py`.

Reply:
I was tempted with that, but it felt like a bit of a hack. I think i'd rather have a tiny breaking change than this hack

Reply:
Why is it a hack? I'd really rather not have to go through all our projects and change this when we upgrade.